modules/amd-installing-gpu-operator.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/amd-gpu-operator.adoc
+
+:_content-type: PROCEDURE
+[id="amd-installing-gpu-operator_{context}"]
+= Installing the AMD GPU Operator
+
+As a cluster administrator, you can install the AMD GPU Operator by using the OpenShift CLI and the web console. Use the following procedure to install the AMD community release of the Operator.
+
+Use the following steps to install and configure the AMD GPU Operator.
+
+. Install the xref:../hardware_enablement/psap-node-feature-discovery-operator.adoc#installing-the-node-feature-discovery-operator_node-feature-discovery-operator[Node Feature Discovery Operator].
+
+. Install the xref:../hardware_enablement/kmm-kernel-module-management.adoc#kmm-install_kernel-module-management-operator[Kernel Module Management Operator].
+
+. Install and configure the link:https://dcgpu.docs.amd.com/projects/gpu-operator/en/main/installation/openshift-olm.html#install-amd-gpu-operator[AMD GPU Operator].
```

Comment:
@egallen @yevgeny-shnaidman Added this link to AMD procedure Erwan provided in the wg-openshift-ai-amd-gpu-ga Slack channel.